Core/src/main/java/com/plotsquared/core/util/StringMan.java

```diff
@@ -320,4 +323,27 @@ public static boolean contains(String name, char c) {
         return col;
     }
 
+    private static final Pattern STRING_SPLIT_PATTERN = Pattern.compile("(?<quoted>\"[\\w ]+\")|(?<single>\\w+)");
+
+    /**
+     * Converts multiple quoted and single strings into a list of strings e.g.
+     * <p>
+     * | Input                                                           | Output                                                                   |
+     * |-----------------------------------------------------------------|--------------------------------------------------------------------------|
+     * | title "sub title"                                               | ["title", "sub title"]                                                   |
+     * | "a title" subtitle                                              | ["a title", "subtitle"]                                                  |
+     * | "title" "subtitle"                                              | ["title", "subtitle"]                                                    |
+     * | "PlotSquared is going well" the authors "and many contributors" | ["PlotSquared is going well", "the", "authors", "and many contributors"] |
+     *
+     * @param message a input string
+     * @return a list of strings
```

Comment:
Please put a since TODO here, and I suggest to use HTML to design the table, the current output is bar useable 
![](https://i.imgur.com/b8ZiNFh.png)